### PR TITLE
[CHANGE] ui option name for feature file boundaries

### PR DESCRIPTION
**Pull request by singjc**
Change peak picking option OSW-PyProphet to Feature File Boundaries

# Contents ([#99](https://github.com/Roestlab/massdash/pull/99))

### Other
- Feature File Peak Boundaries for raw GUI workflow
- Roestlab/massseer into patch/ui_rename_results_peak_boundaries

### Uncategorised!
- [CHANGE] ui option name for feature file boundaries
- [RENAME] feature file peak pickign method

### CODE REVIEW

File: massdash/server/PeakPickingServer.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def __init__(self, peak_picking_settings: PeakPickingUISettings, chrom_plot_sett
 
     def perform_osw_pyprophet_peak_picking(self, xic_data: SqMassLoader, transition_list_ui: Literal['ExtractedIonChromatogramAnalysisUI', 'RawTargetedExtractionAnalysisUI']):
```
Comment (jcharkow) [inline massdash/server/PeakPickingServer.py]: change method name as well?

Comment (jcharkow) [inline massdash/server/PeakPickingServer.py]: I am having difficulties with this method in the GUI could you please look into this? I think that the loader object for the raw data is not currently being passed which causes loadTransitionGroupFeature to fail. Not sure what the best way to solve this without just adding more arguments

**Review by jcharkow — CHANGES_REQUESTED** [review]
Changes are good. However currently there is an issue in the raw peak GUI display failing to find features. Should try to address this in this PR as well because it concerns these. Unsure if we should use the `plotChromatogram` method I added to the loaders or decouple this feature reading from the peak picking server a bit because it is not doing peak picking just reading results and thus needs different arguments from the other peak pickers (e.g. transitionGroupFeatures).

File: massdash/server/PeakPickingServer.py (hunk 1 above)
Comment (singjc) [reply massdash/server/PeakPickingServer.py]: You're getting this error?

> AttributeError: 'NoneType' object has no attribute 'loadTransitionGroupFeature'

Comment (jcharkow) [reply massdash/server/PeakPickingServer.py]: yes thats the error. I think it is because there is no XIC data/loader object passed

Comment (singjc) [reply massdash/server/PeakPickingServer.py]: I passed the mzml loader object now instead to get the feature data from the results file. I think I initially used the sqMass workflow that's why it failed.

**Review by jcharkow — APPROVED** [review]
Good for now however I do think this needs refactoring. I am thinking of separating out FeatureFile from the rest of peak picking because it is inherently different. Will create a separate branch for that though

**Pull request merged**